dd-java-agent/instrumentation/java-lang/src/main/java/datadog/trace/instrumentation/java/lang/StringCallSite.java

```diff
@@ -26,4 +27,64 @@ public static String afterConcat(
     }
     return result;
   }
+
+  @CallSite.After("java.lang.String java.lang.String.toUpperCase()")
+  public static String afterToUppercase(
+      @CallSite.This final String self, @CallSite.Return final String result) {
+    final StringModule module = InstrumentationBridge.STRING;
+    try {
+      if (module != null) {
+        module.onStringToUpperCase(self, result);
+      }
+    } catch (final Throwable e) {
+      module.onUnexpectedException("afetConcat threw", e);
```

Comment:
CopyPasteException 😄 

```suggestion
      module.onUnexpectedException("afterToUppercase threw", e);
```